deltachat-ios/Chat/ChatViewController.swift

```diff
@@ -1693,6 +1696,9 @@ class ChatViewController: UITableViewController, UITableViewDropDelegate {
 
     private func sendSticker(_ image: UIImage) {
         DispatchQueue.global().async { [weak self] in
+            // TODO: Remove this when core is fixed https://github.com/deltachat/deltachat-core-rust/issues/6447
```

Comment:
as said, it is not clear to me if core should scale stickers. discussions before are more that they should just be left alone, the "huge" one iOS-gallery-generated are maybe more, well, an iOS thing

```suggestion
            // stickers may be huge when drag'n'dropped from photo-recognition, scale down to a reasonable size
```